src/TextInput/TextInput.tsx

```diff
@@ -8,6 +9,7 @@ import { CommonFieldTypes } from 'Field/types/commonFieldTypes'
 
 interface Props extends CommonFieldTypes {
   type?: 'text' | 'email' | 'password' | 'time' | 'date'
+  className?: string
```

Comment:
isn't this already part of `CommonFieldTypes`?

Reply:
yup, my bad - fixed in [0c4b15](https://github.com/marshmallow-insurance/smores-react/pull/2045/commits/0c4b158dc1349d31f74ff8d1a31c7446eab400b0)